apps/selvbetjening-backend/src/main/kotlin/jobs/PubliserInntektsjusteringJobb.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.etterlatte.jobs
+
+import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.inntektsjustering.InntektsjusteringService
+import no.nav.etterlatte.kafka.KafkaProdusent
+import no.nav.etterlatte.libs.common.inntektsjustering.Inntektsjustering
+import no.nav.etterlatte.shuttingDown
+import no.nav.etterlatte.toJson
+import no.nav.helse.rapids_rivers.JsonMessage
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+import java.util.Timer
+import java.util.UUID
+import kotlin.concurrent.fixedRateTimer
+
+class PubliserInntektsjusteringJobb(
+    private val produsent: KafkaProdusent<String, String>,
+    private val inntektsjusteringService: InntektsjusteringService,
+) {
+    private val logger = LoggerFactory.getLogger(PubliserInntektsjusteringJobb::class.java)
+
+    fun schedule(): Timer {
+        logger.info("Setter opp ${this.javaClass.simpleName}")
+
+        return fixedRateTimer(
+            name = this.javaClass.simpleName,
+            initialDelay = Duration.of(60, ChronoUnit.MINUTES).toMillis(),
+            period = Duration.of(15, ChronoUnit.MINUTES).toMillis(),
+        ) {
+            runBlocking {
+                if (LeaderElection.isLeader() && !shuttingDown.get()) {
+                    publiserInntektsjusteringer()
+                }
+            }
+        }
+    }
+
+    fun publiserInntektsjusteringer() {
+        runCatching {
+            // fjerner eventuelle duplikater
+            inntektsjusteringService.oppdaterDuplikateInntektsjusteringer(
```

Comment:
Tenker det er bedre å gjøre dette når en ny blir lagret. Du kan legge til en ON CONFLICT UPDATE i inserten som er der nå. 
Se på KlageDao.kt i saksbehandling for eksempel